src/Sulu/Bundle/MediaBundle/Media/ImageConverter/FileImageLoader.php

```diff
@@ -0,0 +1,154 @@
+<?php
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\MediaBundle\Media\ImageConverter;
+
+
+use Imagine\Exception\RuntimeException;
+use Imagine\Image\ImagineInterface;
+use Sulu\Bundle\MediaBundle\Media\Exception\GhostScriptNotFoundException;
+use Sulu\Bundle\MediaBundle\Media\Exception\ImageProxyMediaNotFoundException;
+use Sulu\Bundle\MediaBundle\Media\Exception\InvalidMimeTypeForPreviewException;
+use Sulu\Bundle\MediaBundle\Media\Exception\OriginalFileNotFoundException;
+use Sulu\Bundle\MediaBundle\Media\Video\VideoThumbnailServiceInterface;
+
+/**
```

Comment:
This class is not unit tested, which is quite hard to achieve, because it is directly accessing the file system or executing commands using `shell_exec`.